status/util.py

```diff
@@ -158,7 +160,8 @@ def get(self):
             if not uppmax_ids:
                 break
             project_id = row.value['project'].replace('/', '_')
-            project_nobackup = copy.copy(row.value['project'].split('/')[0])
+            project_id = row.value.get('project', '').replace('/', '_')
+            project_nobackup = copy.copy(project_id.split('/')[0])
```

Comment:
L162 should go away.
L164 used to split on `/` but now it has been replaced with `_` , so I don't think it will work.